lib/libzfs/Makefile.am

```diff
@@ -57,7 +57,7 @@ libzfs_la_LIBADD = \
 	libzutil.la \
 	libuutil.la
 
-libzfs_la_LIBADD += -lm $(LIBCRYPTO_LIBS) $(ZLIB_LIBS) $(LIBFETCH_LIBS) $(LTLIBINTL)
+libzfs_la_LIBADD += -lrt -lm $(LIBCRYPTO_LIBS) $(ZLIB_LIBS) $(LIBFETCH_LIBS) $(LTLIBINTL)
```

Comment:
this will cause issues on macOS and Windows, but they aren't in upstream yet, so you can skip over it. I will add  $(LIBCLOCK_GETTIME) if'en those PRs land.